admin-manual/installation-setup/installation/install-ubuntu.rst

```diff
@@ -157,12 +157,53 @@ Ubuntu 16.04 (Xenial) installation instructions
        sudo service elasticsearch restart
        sudo update-rc.d elasticsearch defaults 95 10
 
-12. Start the remaining services
+12. Check ClamAV signatures (optional). To run the clamav-daemon service, it
+    needs the signature files created by clamav-freshclam to start. The first
+    time clamav-freshclam is started it can sometimes take longer than 1
+    minute.  Please, make sure that the following files have been downloaded at
+    ``/var/lib/clamav`` directory before starting the ClamAV daemon:
+
+    - bytecode.cvd
+    - daily.cld or daily.cvd
+    - main.cvd
```

Comment:
^ they could be inline literals 

```
    - :file:`bytecode.cvd`
    - :file:`daily.cld` or :file:`daily.cvd`
    - :file:`main.cvd`
```

Even better using `:file:` (http://www.sphinx-doc.org/en/stable/markup/inline.html).